Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
I run `go test` manually on my computer.

In my `.travis.yml` file, I use more complex commands: https://github.com/pierrre/githubhook/blob/master/.travis.yml
I think we should run `gotestcover` on a larger project with multiple packages (but only on Travis).